code/mobility-services/mobility-services.js

```
@@ -108,10 +109,50 @@ for (let i = 0; i < textBoxFieldIDs.length; i++) {
 }
 
 /********************************************************/
-/** Relabel Attachment Links in Tables to 'Attachment' **/
+/** Relabel Attachment Links in Tables to Name Field **/
 /********************************************************/
+//  replace attachment file name with name field. hide_name to hide nameField from tabl
+function replaceAttachmentFilenameWithNameField(fileFieldId, nameFieldId, hide_name = true) {
```

@susannegov, overall this PR looks great. One quick question for you - when is this function call with `hide_name = false`?

---

Since the default is true, it will hide the file name column with `hide()`. I added the third parameter since in the Signs and Markings version it did not hide the file name column originally. 

Since in MSP most of the attachments are prepopulated with the insert connected record whereas Signs and Markings the user creates the attachment records.